### ISSUE TITLE
About resizing the mask

### ISSUE THREAD

**Issue by seeclearlyfirst** [naoto0804/pytorch-inpainting-with-partial-conv#35]
Thanks for your sharing！@naoto0804
I found that you generate 512 * 512 masks and resize them to 256 * 256 using transforms.Resize(size=256).
However, the default downsampling method is the linear interpolation,  which causes the value of the resized mask to be not only 0 and 1, but also other values in [0,1]. The resized mask is not exactly the mask defined in the original paper.
Therefore, when I use 256 * 256 mask ( without resizing, i.e. its value only contains 0 and 1 ) for testing the 1000000.pth pre-trained model, the result is not good.
To avoid this, maybe resizing the mask with the nearest neighbor downsampling method when training the network works.

**Comment by naoto0804** [comment]
Wow you are right, thank you so much for the suggestion!

**Comment by yuanlinwang** [comment]
Does this means the 1000000.pth is not suitable for testing any dataset?
